building.md

```diff
@@ -56,9 +56,9 @@ From there, the project will be ready to build right from the Visual Studio UI.
 
 - Generate the project:
   ```cmd
-  cmake -S . -B build-windows -G "Visual Studio 16 2019" -DCMAKE_TOOLCHAIN_FILE=./cmake/toolchain/windows-x64.cmake
+  cmake -S . -B build-windows -G "Visual Studio 17 2022" -DCMAKE_TOOLCHAIN_FILE=./cmake/toolchain/windows-x64.cmake
```

Comment:
explicit generator not necessary. just use whatever VS is installed on the machine.

Reply:
Explicit is request, cmake no works without this

Reply:
Yes it does.